include/NovelRT/Graphics/RenderingService.h

```diff
@@ -0,0 +1,45 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_GRAPHICS_RENDERINGSERVICE_H
+#define NOVELRT_GRAPHICS_RENDERINGSERVICE_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Graphics {
+  class RenderingService {
+  private:
+    bool initialiseRenderPipeline();
+    LoggingService _logger;
+    Windowing::WindowingService* const _windowingService;
```

Comment:
we probably shouldn't cache the windowing service
we do, however, need to know if the window goes down as that invalidates the render service

Reply:
What do you propose as an alternative?

Reply:
Similar to https://github.com/novelrt/NovelRT/pull/98#discussion_r362263945. We should have some way the user can get the current `runner` and be able to get services from that.

The rendering service would then retrieve the window service on construction and subscribe to the resize and close notifications (so it can update itself as appropriate).